README.md

```diff
@@ -17,17 +17,17 @@ pip install -r requirements.txt
 
 ```bash
 # help command
-./dfusion --help
-./dfusion
+./gnop --help
+./gnop
```

Comment:
Yikes.

What about `gprotocol`? `gnoprotcl`? `gnoprotocol`? `gno-cli`? `gp-cli`?

Reply:
hehe, I knewww it

what about just `gno`?

Reply:
everything but `mesa`!

Reply:
if you want short, `gp`?

Reply:
Because `gno` is for whole Gnosis. And this is not for all of Gnosis, only for Gnosis Protocol. In short, `GP`.

Reply:
Ok! I'll use GP, otherwise we can change it anytime in the future too